fc-community-contracts/src/FactChainNFT.sol

```diff
@@ -0,0 +1,59 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.20;
+
+import {ERC721URIStorage} from "./ERC721/extensions/ERC721URIStorage.sol";
```

Comment:
While I understand why you wanted to have the minimal deps here, I think it's not really useful and makes the contract harder to maintain.
It is likely better and easier to keep track of if we simply add a dep on openzepplin

Reply:
*agreed offline* for a next iteration